tree/peg.go

```diff
@@ -639,20 +641,21 @@ func (t *Tree) Compile(file string, args []string, out io.Writer) (err error) {
 				s = s.Complement(t.EndSymbol - 1)
 			case TypeString, TypeCharacter:
 				consumes = true
-				s.Add([]rune(n.String())[0])
+				s.Add(rune(n.String()[0]))
```

Comment:
This will change the behavior of the code. Indexing a string returns a byte not a rune.